speaker/layer2_controller.go

```diff
@@ -28,9 +29,16 @@ import (
 )
 
 type layer2Controller struct {
-	announcer *layer2.Announce
-	myNode    string
-	mList     *memberlist.Memberlist
+	announcer   *layer2.Announce
+	myNode      string
+	mList       *memberlist.Memberlist
+  // Last known value of the active nodes known via memberlist, per service IP.
```

Comment:
indentation is broken in many spots (tab vs space I guess)

Reply:
argh!!  I just did some major changes to my vim setup which messed this up.  :-(

I see the lint job flagged it too.  I'll clean this up ...

Reply:
fixed ...